page-objects/pages/TASKListPage.js

```diff
@@ -16,6 +17,9 @@ const taskTitleContainer = '.assignment.card .title';
 const taskDescriptionContainer = '.assignment .text-muted.ckcontent';
 const taskContainer = '.homework li.card';
 const deleteTaskButtonInPopup = '.delete-modal button.btn-submit';
+const clickWithoutDueDate = "//*[text()='Task19']";
+const ungradedTaskSection = "//div[@class ='v-expansion-panel']//button[@type='button']";
```

Comment:
If you have time in daily i am interest what the effect of // is, to know it in future. :)